sqlglot/dialects/spark.py

```diff
@@ -98,20 +88,22 @@ class Generator(Hive.Generator):
         }
 
         TRANSFORMS = {
-            **{k: v for k, v in Hive.Generator.TRANSFORMS.items() if k not in {exp.ArraySort}},
+            **{k: v for k, v in Hive.Generator.TRANSFORMS.items() if k not in {exp.ArraySort, exp.Split, exp.ILike}},
```

Comment:
remove the split?

Reply:
Hmm I did I think your page hadn't refreshed.